roles/slg.db-backup/templates/galaxy-backup-db.sh.j2

```diff
@@ -141,7 +141,14 @@ set -o nounset
 set +e
 
 echo '  Uploading to swift: '$backup_filename >> $LOGFILE
-swift --quiet --retries 5 --timeout 120 upload --segment-size 2000000000 --segment-threads 1 ${SWIFT_BACKUP_CONTAINER} ${backup_filename} || echo '  swift upload failed for: '$backup_filename >> $LOGFILE
+swift --quiet --retries 5 --timeout 120 upload --segment-size 2000000000 --segment-threads 1 ${SWIFT_BACKUP_CONTAINER} ${backup_filename}
+# if upload was successful create marker file, otherwise log error
+if [ $? -eq 0 ]; then
+    touch ${backup_filename_swift}
+    echo '  created swift marker file: '$backup_filename_swift >> $LOGFILE
+else
+    echo '  swift upload failed for: '$backup_filename >> $LOGFILE
```

Comment:
In the case of error, does this show up in the galaxy-logs slack message?

Reply:
no, but it would be a good idea to have failed swift operations output to slack

Reply:
Thanks @jlqfab . The backups were out of sight out of mind before the disk volume monitoring on grafana, so it's useful to have this info info.